dev/run-benchmarks.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+from sparktestsupport.shellutils import run_cmd
+
+benchmarks = [
+    ['sql/test', 'org.apache.spark.sql.execution.benchmark.AggregateBenchmark'],
```

Comment:
Max, can we do `testOnly` or listing up files in some specific directories so we don't need to add here when we add a new benchmark if this targets to list up all benchmarks?

Reply:
It does `runMain` of specific class. It is recommended way of running benchmarks, look at all benchmarks comments.

Reply:
> listing up files in some specific directories so we don't need to add here when we add a new benchmark if this targets to list up all benchmarks?

I don't like this automation because I usually run sub-set of benchmarks and for me it is easier to just comment unneeded benchmarks in the script.

Reply:
My use case is running some of benchmarks during night, let's say date-time related + parquet/orc. So, I don't need to run all of them because it can take a day to complete. I put other benchmarks (its project and class names) because I had already tested them in the PR  https://github.com/apache/spark/pull/27078